### PR TITLE
Admin: Add input sanitizer as default for log rocket

### PR DESCRIPTION
**Pull request by zhen0**
## Describe this PR
Adds input sanitizer so that new inputs are hidden in log rocket as default.  Also adds data-public to selects and auto-complete which do not contain sensitive data.

### CODE REVIEW

File: src/pages/Flow/Settings/ClockForm.vue [hunk 1]
```diff
@@ -133,14 +133,20 @@ export default {
     confirm() {
       const clockType =
         typeof this[this.clockToAdd] == 'string' ? 'CronClock' : 'IntervalClock'
-      const parameters = this.parameter != null && isValidJson(this.parameter) ? parseJson(this.parameter) : {}
```
Comment (zhen0) [inline src/pages/Flow/Settings/ClockForm.vue:136]: No code change here.  I ran code formatter and it has reformatted this section.

File: src/components/SetStateDialog.vue [hunk 2]
```diff
@@ -82,6 +82,7 @@ export default {
             >
           </v-card-title>
           <v-select
+            data-publicP
```
Comment (stackoverfloweth) [inline src/components/SetStateDialog.vue:85]:
```suggestion
            data-public
```

File: src/pages/Flow/FlowRunHeartbeat-Tile.vue [hunk 3]
```diff
@@ -78,6 +78,7 @@ export default {
   <v-card v-intersect="{ handler: onIntersect }" class="pa-2" tile>
     <CardTitle title="Activity" icon="show_chart" :icon-color="state">
       <v-select
+        data-publicß
```
Comment (stackoverfloweth) [inline src/pages/Flow/FlowRunHeartbeat-Tile.vue:81]:
```suggestion
        data-public
```

**Review by stackoverfloweth — COMMENTED** [review]
I'm unclear how this sanitizes inputs? Does LogRocket use `data-public`?

**Review by stackoverfloweth — APPROVED** [review]

**Pull request merged**